test/e2e/upgrade/upgrade.go

```diff
@@ -226,9 +226,10 @@ func getUpgradeContext(c configv1client.Interface, upgradeImage string) (*upgrad
 	}
 
 	upgradeImages := strings.Split(upgradeImage, ",")
-	if (len(upgradeImages[0]) > 0 && upgradeImages[0] == current.Image) || (len(upgradeImages[0]) > 0 && upgradeImages[0] == current.Version) {
-		return nil, fmt.Errorf("cluster is already at version %s", versionString(*current))
-	}
+	// Commented version check so that final step in paused workers testing could pass
```

Comment:
Turn it into a log message and i think this is ok - the guard is really just there to prevent oopses but i guess it's reasonable to simply verify the final state and the early state.